messenger-app.xcodeproj/project.pbxproj

```diff
@@ -646,13 +652,14 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 59DM9F2W6S;
+				DEVELOPMENT_TEAM = ZAS292B37D;
 				INFOPLIST_FILE = "messenger-app/Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "Messenger-Group.messenger-app";
+				PRODUCT_BUNDLE_IDENTIFIER = com.CornelisHeuperman.Meshenger;
```

Comment:
Does this mean you need to make a change in firebase too?

Reply:
Yes, I do. It's working at the moment but does give a warning. I'll change it before we merge to master.